src/cliActions.js

```diff
@@ -112,7 +113,37 @@ const signingKeyChangePW = ({oldPassword, newPw, newPwConfirmation}, signingKeyF
 
 });
 
+/**
+ * @desc Build bundle json file
+ * @param  {string} pw password of signing key
+ * @param  {string} signingKeyFile signing key file path
+ * @param  {bool} devMode use dev mode to build bundle json file
+ * @return {Promise<Promise>}
+ */
+const dappStreaming = ({ pw }, signingKeyFile, devMode) =>
+  new Promise((res, rej) => {
+    // make sure singing key exist
+    if (!fs.existsSync(signingKeyFile)) {
+      return rej(new Error(`Signing key ("${signingKeyFile}") does not exist`));
+    }
+
+    // read signing key
+    const rawSigningKey = fs.readFileSync(signingKeyFile, 'utf8');
+    const signingKey = JSON.parse(rawSigningKey);
+
+    // first decrypt the old signing key
```

Comment:
`// first decrypt the old signing key` looks like a copy / past mistake from the change password command

Reply:
done